### PR TITLE
added RedHeadphone Profile

### PR DESCRIPTION
**Pull request by RedHeadphone**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1875"></a>

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉

File: public/data/RedHeadphone.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        "url": "https://codeforces.com/profile/redheadphone",
+        "icon": "codeforces"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/huzaifa-khilawala/",
+        "icon": "linkedin"
+      }
+    ],
+    "milestones": [
```
Comment (kumarsonsoff3) [inline public/data/RedHeadphone.json:33]:
```suggestion
```

File: public/data/RedHeadphone.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        "name": "Friend me on Codeforces",
+        "url": "https://codeforces.com/profile/redheadphone",
+        "icon": "codeforces"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/huzaifa-khilawala/",
+        "icon": "linkedin"
+      }
+    ],
```
Comment (kumarsonsoff3) [inline public/data/RedHeadphone.json:32]:
```suggestion
```

**Review by kumarsonsoff3 — DISMISSED** [review]
I recommend removing line no. `32`, `33`, as you're not using the milestone section!

So it will be good to remove the fields not in use! :)

**Review by Panquesito7 — APPROVED** [review]

**Review by EmmaDawsonDev — APPROVED** [review]

**Pull request merged**